### PR TITLE
Add AAP on AWS min required roles

### PR DESCRIPTION
**Pull request by zkayyali812**
To deploy foundation

### CODE REVIEW

File: stories/topics/ref-aap-aws-install-prerequisites.adoc [hunk 1]
```diff
@@ -27,8 +27,61 @@ You must have a working knowledge of the following aspects of {AWS}:
 
 For more information about {AWS} and terminology, see the link:https://aws.amazon.com/[AWS product documentation].
 
-Your AWS account must have _Identity and Access Management_ (IAM) permissions to create and manage {Market} deployments as well as the resources described in xref:con-aws-application-architecture[Application architecture]. 
-The application can fail to deploy if your IAM policies restrict deployment and management of these resources.
+Your AWS account must have _Identity and Access Management_ (IAM) permissions below to create and manage {Market} deployments as well as the resources described in xref:con-aws-application-architecture[Application architecture]. Your AWS account must also be licensed to deploy {PlatformNameShort} from AWS Marketplace.
```
Comment (ianf77) [inline stories/topics/ref-aap-aws-install-prerequisites.adoc:30]: {PlatformNameShort} from AWS Marketplace is now {AAPonAWS} much easier to type!

File: stories/topics/ref-aap-aws-install-prerequisites.adoc [hunk 2]
```diff
@@ -27,8 +27,61 @@ You must have a working knowledge of the following aspects of {AWS}:
...
+*  Managed Policies
+** AWSCloudFormationReadOnlyAccess
+** AmazonS3ReadOnlyAccess
+** IAMReadOnlyAccess
+** AmazonRDSReadOnlyAccess
+** AmazonElasticFileSystemReadOnlyAccess
+** AmazonEC2FullAccess
+** AWSMarketplaceFullAccess
+* Cloudformation inline IAM policies
+** cloudformation:DeleteStack
```
Comment (ianf77) [inline stories/topics/ref-aap-aws-install-prerequisites.adoc:44]: Is it Cloudformation or cloudformation?

and below

**Review by ianf77 — COMMENTED** [review]
A couple of comments

File: stories/topics/ref-aap-aws-install-prerequisites.adoc (hunk 2 above)
Comment (zkayyali812) [reply stories/topics/ref-aap-aws-install-prerequisites.adoc:44]: The official IAM policy is lowercase, but CloudFormation is the product name

**Review by ianf77 — APPROVED** [review]
LGTM

**Comment by ianf77** [comment]
Do you  want this merged?

**Comment by zkayyali812** [comment]
@ianf77 Yes please

**Pull request merged**